src/connector/src/sink/iceberg/mod.rs

```diff
@@ -152,6 +152,9 @@ pub struct IcebergConfig {
 
     #[serde(default, deserialize_with = "deserialize_bool_from_string")]
     pub create_table_if_not_exists: bool,
+
+    #[serde(default, deserialize_with = "deserialize_bool_from_string")]
```

Comment:
just a question: Is there a situation where we create non-nimtable services in a branch?

This config item is a Bool and I think it should have corresponding prefixes like ‘is_xxx’, ‘enable_xxx’.... .etc

Reply:
Good point. I think using `enable_config_load` is a better name.